ui/components/FormDialog.vue

```diff
@@ -49,7 +50,7 @@ const props = defineProps({
     type: String,
     required: true,
     validator(value) {
-      return ['wizard', 'edit'].includes(value);
+      return Object.values(FORM_DIALOG_TYPES_DICT).includes(value);
```

Comment:
@nerimartinez Turns out that, even though we cannot use local variables in compiler macros, we can reference imported bindings :)

See https://vuejs.org/api/sfc-script-setup.html#defineprops-defineemits:~:text=it%20can%20reference%20imported%20bindings